uint8[] color
uint8[] depth_upper
uint8[] depth_lower
float64[] color_info
float64[] depth_info 
